tripal_chado/tests/src/Kernel/ChadoCustomTables/ChadoMviewTest.php

```diff
@@ -0,0 +1,285 @@
+use Drupal\tripal_chado\ChadoCustomTables\ChadoMview;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\Attributes\Group;
+use PHPUnit\Framework\Attributes\RunTestsInSeparateProcesses;
+use PHPUnit\Framework\Attributes\Test;
+
+/**
+ * Kernel test for ChadoMview create/populate/delete lifecycle.
+ *
+ * @group tripal-chado
+ * @group service-chado-custom-table
+ */
+#[CoversClass(ChadoMview::class)]
+#[Group('tripal-chado')]
+#[Group('service-chado-custom-table')]
+#[RunTestsInSeparateProcesses]
+class ChadoMviewTest extends ChadoTestKernelBase {
+
+  /**
+   * Modules to enable for this Kernel test.
+   *
+   * @var array
+   */
+  protected static $modules = [
+    'tripal',
+    'tripal_chado',
+  ];
+
+  /**
+   * Connection to Drupal public schema.
+   *
+   * @var Drupal\Core\Database\Connection
+   */
+  protected Connection $drupal_connection;
+
+  /**
+   * Connection to Chado schema.
+   *
+   * @var Drupal\tripal_chado\Database\ChadoConnection
+   */
+  protected ChadoConnection $chado_connection;
+
+  /**
+   * Record IDs of chado organisms.
+   *
+   * @var array
+   */
+  protected array $organism_ids;
+
+  /**
+   * The most recent error message from the mocked tripal logger.
+   *
+   * @var string
+   *   The error message.
+   */
+  protected string $mock_error = '';
+
+  /**
+   * Test initialization, get database connections.
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    // Drupal DB connection for metadata assertions.
+    $this->drupal_connection = \Drupal::database();
+
+    // Initialize a prepared Chado schema (contains baseline data).
+    // PREPARE_TEST_CHADO includes records loaded during "prepare".
+    $this->chado_connection = $this->getTestSchema(self::PREPARE_TEST_CHADO);
+
+    // Install required schemas.
+    $this->installSchema('tripal_chado', ['tripal_custom_tables', 'tripal_mviews']);
+
+    // Capture tripal logger errors.
+    $mock_logger = $this->getMockBuilder(TripalLogger::class)
+      ->onlyMethods(['error'])
+      ->getMock();
+    $mock_logger->method('error')
+      ->willReturnCallback(function ($message, $context, $options) {
+          $this->mock_error .= str_replace(array_keys($context), $context, $message);
+          return NULL;
+      });
+    \Drupal::getContainer()->set('tripal.logger', $mock_logger);
+
+    // Create organism records, common name length is 19 characters.
+    $this->organism_ids[] = $this->chado_connection
+      ->insert('1:organism')
+      ->fields(['genus' => 'Tripalus', 'species' => 'bogusii', 'common_name' => 'Common false tripal'])
+      ->execute();
+    $this->organism_ids[] = $this->chado_connection
+      ->insert('1:organism')
+      ->fields(['genus' => 'Tripalus', 'species' => 'databasica'])
+      ->execute();
+  }
+
+  /**
+   * Kernel test for ChadoMview create/populate/delete lifecycle.
+   */
+  #[Test]
+  public function createPopulateDeleteMviewTest(): void {
```

Comment:
Switch to the standard testTESTNAME format for the method name so that `#[Test]` is not required.

```suggestion
  public function testCreatePopulateDeleteMview(): void {
```